style.css

```diff
@@ -118,3 +115,228 @@ ul {
   left: 128px;
   top: 111px;
 }
+
+/* work section */
+.work {
+  display: grid;
+  width: 95%;
+  margin-top: 15%;
+  grid-template-rows: 650px 700px 650px 700px;
+  padding: 50px 10px 10px 10px;
+  background-color: #e5e5e5;
+  row-gap: 100px;
+}
+
+.project {
+  background-color: white;
+  margin-top: 10px;
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100%;
+  padding: 19px;
+  color: white;
+  text-align: center;
+  border-radius: 10px;
+  border: 1px solid #e0e0e0;
+}
+
+.screenshot > img {
+  width: 100%;
+}
+
+.project-name {
+  text-align: left;
+  margin-top: 5px;
+  font-style: normal;
+  font-weight: bolder;
+  font-size: 42px;
+  line-height: 44px;
+  color: #172b4d;
+  text-transform: capitalize;
+}
+
+.meta {
+  margin-top: -33px;
+  display: flex;
+  justify-content: flex-start;
+  font-size: 13px;
+}
+
+.meta > * {
+  padding-right: 10px;
+}
+
+.meta > img {
+  width: 8px;
+  height: 8px;
+  align-self: center;
+  margin-top: -3px;
+}
+
+.social > li > img {
+  height: 25px;
+}
+
+.meta > :first-child {
+  text-transform: uppercase;
+  color: #344563;
+  font-weight: bolder;
+}
+
+.meta > :nth-child(3) {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: capitalize;
+  color: #7a869a;
+}
+
+.meta > :nth-child(5) {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: capitalize;
+  color: #7a869a;
+}
+
+.second-text {
+  margin-top: -3px;
+  color: #344563;
+  text-align: left;
+}
+
+.tools {
+  display: flex;
+  justify-content: space-between;
+  width: 80%;
+  margin: 0;
+  padding: 0;
+  margin-top: -10px;
+}
+
+.tools > li {
+  padding: 0 12px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 8px;
+}
+
+.button {
+  display: block;
+  text-decoration: none;
+  background-color: white;
+  color: #396df2;
+  border: 1px solid #396df2;
```

Comment:
- [ ] **OPTIONAL** The styling of the bottom of each project's card needs a few modifications if you got time for it.
> I'm sorry this didn't show up for you in the earlier review.

However, it'd be great if you followed the given design to complete this amazing website 🥰

_Please look at the images down below._
### **Your design**
![Screenshot1](https://user-images.githubusercontent.com/47625407/120877134-1363eb80-c5b5-11eb-95fd-123230fdc5e8.png)
### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A546)**
![Screenshot](https://user-images.githubusercontent.com/47625407/120877150-25458e80-c5b5-11eb-9731-a089841dbfda.png)